torch/quantization/_equalize.py

```diff
@@ -38,16 +39,18 @@ def cross_layer_equalization(module1, module2, output_axis=0, input_axis=1):
     the ranges of the first tensors' output channel are equal to the
     ranges of the second tensors' input channel
     '''
-    if type(module1) not in _supported_types or type(module2) not in _supported_types:
+    if (type(module1) not in _supported_types or type(module2) not in _supported_types) and \
+       (type(module1) not in _supported_intrinsic_types or type(module2) not in _supported_intrinsic_types):
```

Comment:
can you combine _supported_types and _supported_intrinsic_types in the same set and do the original check?
```
if type(module1) not in _all_supported_types or type(module2) not in _all_supported_types:
```
I think the current check will miss something

Reply:
Could I create another global `_all_supported_types` which is the union of `_supported_types` and `_supported_intrinsic_types`? I wanted to keep `_supported_types` and `_supported_intrinsic_types` separate so that we could case on them when getting the weights. Would this be ok?